images/win/scripts/Installers/Install-BizTalkBuildComponent.ps1

```diff
@@ -0,0 +1,89 @@
+################################################################################
+##  File:  Install-BizTalkBuildComponent.ps1
+##  Desc:  Install BizTalk Project Build Component
+################################################################################
+
+function Install-Msi
+{
+    <#
+    .SYNOPSIS
+        A helper function to install executables.
+
+    .DESCRIPTION
+        install .exe or .msi binaries from specified Path.
+
+    .PARAMETER MsiPath
+        Msi or exe path. Required parameter.
+
+    .PARAMETER LogPath
+        The log file path where installation will write log to. Required parameter.
+
+    .EXAMPLE
+        Install-Msi -MsiPath "c:\temp\abc.msi" -LogPath "c:\abc.log"
+    #>
+
+    Param
+    (
+        [Parameter(Mandatory)]
+        [String] $MsiPath,
+        [Parameter(Mandatory)]
+        [String] $LogPath
+    )
+
+    try
+    {
+        $filePath = "msiexec.exe"
+
+        Write-Host "Starting Install $MsiPath..."
+        $ArgumentList = ('/i', $MsiPath, '/QN', '/norestart', "/l*v",$LogPath)
+        $process = Start-Process -FilePath $filePath -ArgumentList $ArgumentList -Wait -PassThru -Verb runAs
+
+        $exitCode = $process.ExitCode
+        if ($exitCode -eq 0 -or $exitCode -eq 3010)
+        {
+            Write-Host "Installation for $MsiPath is successful."
+        }
+        else
+        {
+            Write-Host "Non zero exit code returned by $MsiPath installation process: $exitCode"
+            Get-Content  $LogPath | Write-Host
+            exit $exitCode
+        }
+    }
+    catch
+    {
+        Write-Host "Failed to install $MsiPath : $($_.Exception.Message)"
+        exit 1
+    }
+}
+
+$bizTalkBuildComponentUri = "https://aka.ms/BuildComponentSetup.EN"
+
+# Download
+try {
```

Comment:
No need to use try-catch, the same logic already exists in the function  `Start-DownloadWithRetry` -    https://github.com/actions/virtual-environments/blob/d511b0d2da53ec9a1fb9af3224d55607240da7b6/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L190

Reply:
Resolved.